src/types/course/types/course.types.ts

```diff
@@ -1,6 +1,13 @@
-import { Lesson, Quiz, Attachment, ItemsWithCount } from '~/types'
-import { Course } from '~/types/course/interfaces/course.interface'
+import {
+  Lesson,
+  Quiz,
+  Attachment,
+  ResourceAvailabilityStatusEnum
+} from '~/types'
 
-export type CourseResources = Lesson | Quiz | Attachment
+type Resources = Lesson | Quiz | Attachment
 
-export type MyCorsesListProps = Omit<ItemsWithCount<Course>, 'count'>
+export type CourseResources = Resources & {
```

Comment:
here better use interface extends
like 
`interface CourseResources extends Resources` and add your fields

Reply:
It's type, actually

Reply:
> It's type, actually

I told you about interface

FYI
<img width="687" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/ca7f6df1-2fae-4b73-a23a-4088f42381db">

Reply:
`Resources` is not a static type, so an interface cannot extend from it. 

![telegram-cloud-photo-size-4-5881759469522764621-y](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/108689551/4bd128b4-4a8d-4c3c-818d-26c63b939676)


Extending interfaces of Lesson, Quiz and Attachment does not look good to me, since we will have to do that in three separate files and those interfaces are used not only for the Resources type. Therefore it makes sense to only apply these additional fields here.